test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,573 @@
+	vpc := vpcs[0]
+	foundCidr := aws.StringValue(vpc.CidrBlock)
+	if foundCidr != expectedCidr {
+		newErr.vpcError = append(newErr.vpcError, fmt.Sprintf("cidr blocks not equal, expected %s but got %s", expectedCidr, foundCidr))
+	}
+
+	return vpc, newErr
+}
+
+// verify that the vpc subnets are created
+func verifySubnets(session *ec2.EC2, clusterTag, expectedCidr string) ([]*ec2.Subnet, error) {
+	newErr := &networkConfigTestError{
+		subnetsError: []string{},
+	}
+
+	// filter subnets by name and integreatly tag
+	describeSubnets, err := session.DescribeSubnets(&ec2.DescribeSubnetsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:Name"),
+				Values: []*string{aws.String(subnetTagName)},
+			},
+			{
+				Name:   aws.String("tag:integreatly.org/clusterID"),
+				Values: []*string{aws.String(clusterTag)},
+			},
+		},
+	})
+	if err != nil {
+		newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("could not describe subnets: %v", err))
+		return nil, newErr
+	}
+
+	// parse the vpc cidr block from the createStrategy for _network
+	subnets := describeSubnets.Subnets
+	_, cidr, err := net.ParseCIDR(expectedCidr)
+	if err != nil {
+		newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("could not parse vpc cidr block: %v", err))
+		return subnets, newErr
+	}
+	cidrMask, _ := cidr.Mask.Size()
+
+	// verify the subnet masks for the subnets are one bit bigger
+	// than the vpc subnet mask
+	for _, subnet := range subnets {
+		_, subnetCidr, err := net.ParseCIDR(aws.StringValue(subnet.CidrBlock))
+		if err != nil {
+			newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("could not parse subnet mask for vpc subnets: %v", err))
+			return subnets, newErr
+		}
+		subnetCidrMask, _ := subnetCidr.Mask.Size()
+		if subnetCidrMask != cidrMask+1 {
+			newErr.subnetsError = append(newErr.subnetsError, fmt.Sprintf("unexpected subnet mask size for vpc subnets: %v", err))
+		}
+	}
+
+	return subnets, newErr
+}
+
+// verify vpc security group
+func verifySecurityGroup(session *ec2.EC2, clusterTag string) error {
+	newErr := &networkConfigTestError{
+		securityGroupError: []string{},
+	}
+
+	// filter security groups by name and integreatly tag
+	describeGroups, err := session.DescribeSecurityGroups(&ec2.DescribeSecurityGroupsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:Name"),
+				Values: []*string{aws.String(securityGroupTagName)},
+			},
+			{
+				Name:   aws.String("tag:integreatly.org/clusterID"),
+				Values: []*string{aws.String(clusterTag)},
+			},
+		},
+	})
+	if err != nil {
+		newErr.securityGroupError = append(newErr.securityGroupError, fmt.Sprintf("could not find vpc security group: %v", err))
+	}
+
+	// expect 1 security group
+	secGroups := describeGroups.SecurityGroups
+	if len(secGroups) != 1 {
+		newErr.securityGroupError = append(newErr.securityGroupError, fmt.Sprintf("unexpected number of security groups: %d", len(secGroups)))
+		return newErr
+	}
+
```

Comment:
Should we be validating the ingress rules here, or is it okay to valid the security group exists? 
Goes to what we talked about earlier, if the connectivity metrics fail, it would point to this.